.gitlab-ci.yml

```
@@ -159,6 +169,19 @@ default:
     # this job runs only on nightly pipeline with the mentioned variable, against `master` branch
     - if: $CI_COMMIT_REF_NAME == "master" && $CI_PIPELINE_SOURCE == "schedule" && $PIPELINE == "nightly"
 
+.build-rustdoc-rules:              &build-rustdoc-rules
+  rules:
+    - if: $CI_PIPELINE_SOURCE == "pipeline"
+      when: never
+    - if: $CI_PIPELINE_SOURCE == "web" && $RM_RUSTDOC_REF
```

```suggestion
    - if: $CI_PIPELINE_SOURCE == "schedule" && $RM_RUSTDOC_REF
```

It's better to create some preset schedules https://gitlab.parity.io/parity/substrate/-/pipeline_schedules with variables and branches and set them to manual (not active). And explain it in the attached doc. Also it's possible to run just rusdocs from those "manual schedules" jobs without running all the other CI.